src/Database/Traits/Sortable.php

```diff
@@ -75,6 +75,6 @@ public function setSortableOrder($itemIds, $itemOrders = null)
      */
     public function getSortOrderColumn()
     {
-        return defined('static::SORT_ORDER') ? static::SORT_ORDER : 'sort_order';
+        return defined('static::SORT_ORDER') ? constant('static::SORT_ORDER') : 'sort_order';
```

Comment:
Why?

Reply:
Both PHPStan AND Intellisense/Intelephense report it as a missing constant, even with the `defined` check surrounding it.

Reply:
Silly automated tools...